modules/rtps/optical_flow_Subscriber.cpp

```diff
@@ -0,0 +1,128 @@
+/****************************************************************************
```

Comment:
@avinash-palleti Why do we need an optical flow subscriber?

Reply:
No need. I will remove both header and source.

Reply:
Thx!:+1: